src/main/java/com/uid2/shared/store/IKeysetKeyStore.java

```diff
@@ -11,7 +11,7 @@ public interface IKeysetKeyStore {
 
 
     public interface IkeysetKeyStoreSnapshot {
```

Comment:
why does this interface exist - can we remove it?

Reply:
This is how all the other snapshots work

Reply:
bit harder to fix those ones though. We can fix this one and fix those later

Reply:
LGTM other than this nested interface IkeysetKeyStoreSnapshot. Can we remove this, replace all instances with KeysetKeyStoreSnapshot, and then it's good to merge

Reply:
I don't think this work is really in scope for the this story, we should do it on another card